react/src/lib/SpaceListMeeting/index.js

```diff
@@ -103,9 +103,10 @@ class SpaceListMeeting extends React.PureComponent {
               direction="bottom-center"
               targetOffset={{ vertical: 3 }}
               isDynamic
+              includeFocusOnHover={false}
               {...eventOverlayProps}
             >
-              <span role="button" tabIndex={0} className="md-list-item--space-meeting--attendees">
+              <span role="button" tabIndex={-1} className="md-list-item--space-meeting--attendees">
```

Comment:
Assuming this is desired? @aimeex3

Reply:
If we don't make it initially -1, then we can tab through all of them in the list.

Reply:
We need to do it for my follow up PR on listitems